pkg/api/domain.go

```diff
@@ -26,6 +26,8 @@ const (
 	QuayOpenShiftCIRepo = "quay.io/openshift/ci"
 
 	QCIAPPCIDomain = "quay-proxy.ci.openshift.org"
+
+	QCICache = "qci-pull-through-cache-us-east-1-ci.apps.ci.l2s4.p1.openshiftapps.com/openshift/ci"
```

Comment:
```suggestion
	QCICacheDomain = "qci-pull-through-cache-us-east-1-ci.apps.ci.l2s4.p1.openshiftapps.com"
```

We will modify https://github.com/openshift/release/blob/master/core-services/ci-secret-bootstrap/_config.yaml#L692 accordingly.

I missed it in https://github.com/openshift/release/pull/52732/files (or I cannot remember the reason now)